packages/pfsense/data_stream/log/elasticsearch/ingest_pipeline/haproxy.yml

```
@@ -123,6 +128,11 @@ processors:
       field: event.outcome
       value: failure
       if: "ctx?.http?.response?.status_code != null && ctx.http.response.status_code >= 400"
+  - remove:
+      field: 
+        - temp.duration
```

```suggestion
        - temp
```

But I think it should be created as `_temp` in the grok patterns to make it more obviously out of place.